common/errors.h

```diff
@@ -212,6 +212,8 @@ enum retcode_t {
   RC_CCLIENT_RES_ERROR = 0x07 | RC_MODULE_CCLIENT | RC_SEVERITY_MODERATE,
   // json parsing error, key no found in the json object
   RC_CCLIENT_JSON_KEY = 0x08 | RC_MODULE_CCLIENT | RC_SEVERITY_MAJOR,
+  RC_CCLIENT_UTLIST = 0x09 | RC_MODULE_CCLIENT | RC_SEVERITY_MAJOR,
+  RC_CCLIENT_FLEX_TRITS = 0x0A | RC_MODULE_CCLIENT | RC_SEVERITY_MAJOR,
```

Comment:
`RC_CCLIENT_FLEX_TRITS/RC_CCLIENT_UTLIST/RC_CCLIENT_JSON_KEY` do not indicate something went wrong

Reply:
`RC_CCLIENT_FLEX_TRITS` indicates trits converting error. 
`RC_CCLIENT_JSON_KEY` caused by unexpected JSON object operation between IRI and client. it indicates communication error.
both should not happen at runtime.

Reply:
i mean that the name should say it's error, like `RC_CCLIENT_JSON_KEY` -> `RC_CCLIENT_JSON_NULL_KEY` or `RC_CCLIENT_JSON_INVALID_KEY`,
but ok